manifests/plugin/remote_execution.pp

```diff
@@ -1,7 +1,57 @@
+# = Foreman Remote Execution
 # Installs foreman_remote_execution plugin
-class foreman::plugin::remote_execution {
+#
+# === Parameters:
+#
+# $cockpit:: Install cockpit support
+#
+class foreman::plugin::remote_execution (
+  Boolean $cockpit = false,
+) {
   include ::foreman::plugin::tasks
+  include apache::mod::proxy_wstunnel
+  include apache::mod::proxy_http
 
   foreman::plugin {'remote_execution':
   }
+
+  $cockpit_subpath = 'webcon'
+
+  if $cockpit {
+    if $::osfamily != 'RedHat' {
```

Comment:
I'm a bit on the fence about this. Is there a reason why it can't work on Debian?